cmd/ci-secret-bootstrap/main.go

```diff
@@ -512,7 +517,19 @@ func updateSecrets(secretsGetters map[string]coreclientset.SecretsGetter, secret
 			logger := logger.WithFields(logrus.Fields{"namespace": secret.Namespace, "name": secret.Name, "type": secret.Type})
 			logger.Debug("handling secret")
 
-			secretClient := secretsGetters[cluster].Secrets(secret.Namespace)
+			nsClient := getters[cluster].Namespaces()
+			if _, err := nsClient.Get(context.TODO(), secret.Namespace, metav1.GetOptions{}); err != nil {
```

Comment:
IMHO we can just try to create the namespace and if it already exists, we can eat the error?

Reply:
We can. Why is it better?

Reply:
fewer API calls.